x-pack/plugins/security_solution/public/timelines/components/notes/add_note/index.tsx

```diff
@@ -84,6 +87,10 @@ export const AddNote = React.memo<{
     [onCancelAddNote]
   );
 
+  const isAddNoteDisabled = useMemo(() => {
+    return newNote.trim().length === 0 && authenticatedUser !== null;
```

Comment:
Shouldn't this be that the note is disabled if the `authenticatedUser == null` ?

Reply:
@michaelolo24 I removed this check entirely, was also what was causing tests to fail ha

Reply:
Sweet, thanks!